package.json

```diff
@@ -8,7 +8,9 @@
     "@material-ui/core": "^4.8.3",
     "@material-ui/icons": "^4.5.1",
     "@material-ui/lab": "^4.0.0-alpha.40",
+    "c3": "^0.7.12",
     "csvtojson": "^2.0.10",
+    "d3": "^5.15.0",
```

Comment:
Should we remove `"plotly.js` from package.json as well?

Reply:
Sure, but that doesn't really affect on performance.
And the yarn.lock changes on that is quite large